screens/ShoppingList.js

```diff
@@ -0,0 +1,139 @@
+
+import React, { Component } from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, AppRegistry, FlatList } from 'react-native';
+
+let colors = ['#6DC4E0', '#605DF1']
+
+let items = [
+  {
+    name: 'Name',
+    TimeLocation: 'Time, Location',
```

Comment:
Don't use Pascal Case for variables. Use Camel Case for keys in dictionaries, and variables in general